config/config.txt

```diff
@@ -499,7 +499,7 @@ MINUTE_CLICK_LIMIT 500
 #ERROR_MSG_DELAY 50
 
 ## Send a message to IRC when starting a new game
-#IRC_ANNOUNCE_NEW_GAME
```

Comment:
Actually, this is entirely incorrect. Our code currently uses this as a string value for what channel (defined within TGS) to send this to.
It's not actually able to be disabled, nor is the commented out case handled.

Reply:
I don't understand why the config code states it's deprecated and states it should be removed then, a downstream of NSV recently tried to use IRC_ANNOUNCE and it didn't work because having a the deprecationvalue return text (`""` is text) sets the value of the config entry to that upon loading.
Code for the entry:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/43698041/42993831-1792-494a-b3d1-81f7fcebb3c3)
Deprecated config entry when loading:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/43698041/94beb36d-8256-4eba-9171-62eea6447a0a)

Reply:
No, the new entry. Chat announce.